$OpenBSD: patch-giscanner_utils_py,v 1.2 2011/04/07 11:09:06 ajacoutot Exp $

Make sure try to compile/link with the right libtool,
this overrides the --libtool option to girscanner.

--- giscanner/utils.py.orig	Tue Sep  7 16:25:05 2010
+++ giscanner/utils.py	Thu Apr  7 12:52:18 2011
@@ -107,22 +107,8 @@ def get_libtool_command(options):
     if not libtool_infection:
         return None
 
-    libtool_path = options.libtool_path
-    if libtool_path:
-        # Automake by default sets:
-        # LIBTOOL = $(SHELL) $(top_builddir)/libtool
-        # To be strictly correct we would have to parse shell.  For now
-        # we simply split().
-        return libtool_path.split(' ')
-
-    try:
-        subprocess.check_call(['libtool', '--version'],
-                              stdout=open(os.devnull))
-    except (subprocess.CalledProcessError, OSError), e:
-        # If libtool's not installed, assume we don't need it
-        return None
-
-    return ['libtool']
+    libtool_cmd = os.environ.get('LIBTOOL', '/usr/ports/infrastructure/bin/libtool')
+    return libtool_cmd.split()
 
 
 def files_are_identical(path1, path2):
